src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
What is changed here?

Reply:
I think that you've changed the BOM on this file.

Reply:
Github seems to be incorrectly showing a diff, when none exists. We have been ignoring it for all PRs, doing the same for this one.

Reply:
You have indeed changed (removed) the BOM.

![image](https://user-images.githubusercontent.com/953151/30821151-373407ec-a1d9-11e7-9e03-72af7ec4167b.png)

Reply:
Sorry, got my diff backwards - you added the BOM (which is correct style for roslyn code, iirc)

Reply:
@TyOverby @khyperia - I just opened the file in VS to edit. @333fred Is it likely that our Operation unit test baseline updater is changing the BOM?

Reply:
I am merging in as the PR seems to be adding the correct BOM.

Reply:
@mavasani probably, I was just using File.WriteAllText with no encoding specifier. I'll update it to use the existing encoding.

Reply:
@333fred You might be interested in the patch that I made correcting BOM encoding in my autoformatter.

```diff
diff --git a/Program.cs b/Program.cs
index 4654046..059d0bc 100644
--- a/Program.cs
+++ b/Program.cs
@@ -8,6 +8,7 @@
 using static Microsoft.CodeAnalysis.Formatting.Formatter;
 using Microsoft.CodeAnalysis.Text;
 using System.Diagnostics;
+using System.Text;
 
 namespace CSharpFormatter
 {
@@ -86,6 +87,7 @@ static void ApplyChangesToDocument(string documentPath, IEnumerable<DiffRequest>
         {
             if (documentPath.EndsWith("vb")) { return; }
             SyntaxNode after;
+            Encoding encoding;
             using (var file = File.Open(documentPath, FileMode.Open))
             {
                 var sourceText = SourceText.From(file);
@@ -96,10 +98,12 @@ static void ApplyChangesToDocument(string documentPath, IEnumerable<DiffRequest>
                 // need to be able to adjust them.
                 requests = requests.OrderBy(r => r.lineStart);
                 var spans = requests.Select(req => SpanForLines(sourceText, req.lineStart, req.lineCount));
+
                 after = Format(tree.GetRoot(), spans, new AdhocWorkspace());
+                encoding = sourceText.Encoding;
             }
 
-            File.WriteAllText(documentPath, after.ToFullString());
+            File.WriteAllText(documentPath, after.ToFullString(), encoding);
         }
 
         static IEnumerable<string> ConsoleLines()

```

Reply:
I would indeed, I was just figured out how to get the current file encoding.

Reply:
@mavasani pushed an update to my fixer that uses the correct file encoding.

Reply:
Thanks Fred.

Reply:
@333fred Could you run this fixer over all of roslyn in order to correct any other files that may have been mistakenly switched?

Reply:
@TyOverby this fixer isn't designed for that, it scans for IOperation tests that need to have their baselines updated. Your formatter with the process all roslyn switch is much closer.

Reply:
@333fred My formatter won't add the bom on files that don't have it.  It'll correctly write it back the way it was written.